jupyter_collaboration/handlers.py

```diff
@@ -321,6 +331,14 @@ def on_close(self) -> None:
             self.log.info("Cleaning room: %s", self._room_id)
             self.room.cleaner = asyncio.create_task(self._clean_room())
 
+    def _emit(self, action: str, msg: str, warn: bool = False) -> None:
```

Comment:
Here we should switch to an enum:

```suggestion
    def _emit(self, action: str, msg: str, level: Enum = INFO) -> None:
```